loader/db.go

```diff
@@ -278,3 +296,258 @@ func isMySQLError(err error, code uint16) bool {
 	e, ok := err.(*mysql.MySQLError)
 	return ok && e.Number == code
 }
+
+func processErrSQL(ctx *tcontext.Context, conn *Conn, txn *sql.Tx, sql string) {
+	var (
+		dupData     string
+		table       string
+		messages    []string
+		tmpDupData  string
+		keyName     string
+		dupDataPair string
+		flag        bool
+		query       string
+	)
+
+	reg := regexp.MustCompile(`^INSERT IGNORE INTO .*?VALUES`)
+	if reg.MatchString(sql) {
+		tmpTable := reg.FindAllString(sql, 1)
+		table = tmpTable[0][20 : len(tmpTable[0])-8]
+	} else {
+		ctx.L().Warn("get duplicate entry", zap.Error(errors.New("Can't get table name")))
+		return
+	}
+
+	processTmpErr := func(mess string, err error) {
+		ctx.L().Warn("get duplicate entry", zap.String("dupEntry information", mess), zap.Error(err))
+	}
+
+	warns, err := tutils.ShowWarnings(txn)
+	if err != nil {
+		ctx.L().Warn("get duplicate entry", zap.Error(err))
+		return
+	}
+
+	for i := 0; i < len(warns); i++ {
+		if warns[i].Level == "Warning" && warns[i].Code == "1062" {
```

Comment:
it is better to use mysql error code https://github.com/pingcap/parser/blob/2bb2c0e8f34025aa5c5ceb713ec5233936e3d069/mysql/errcode.go#L82